lib/generators/rolemodel/mailers/templates/app/javascript/packs/mailer_stylesheets.scss

```diff
@@ -0,0 +1,44 @@
+@import 'stylesheets/vendor/normalize';
+@import 'stylesheets/theme';
+@import 'stylesheets/variables';
+@import 'stylesheets/base';
+@import 'stylesheets/utilities';
+
+@import 'stylesheets/components/flash';
+@import 'stylesheets/components/button';
+@import 'stylesheets/components/card';
+@import 'stylesheets/components/form';
+@import 'stylesheets/components/table';
+
+@import 'stylesheets/components/icon';
+
+:root {
+  --neutral-layer-app-background: #{$color-neutral-50};
```

Comment:
These Scss style variables are going away with some of the color theory changes were are making